src/main/java/com/flansmod/client/model/ModelVehicle.java

```diff
@@ -34,6 +45,9 @@
 	public ModelRendererTurbo drillHeadModel[] = new ModelRendererTurbo[0]; 		//Drill head. Rotates around
 	public Vector3f drillHeadOrigin = new Vector3f();								//this point
 	
+	public float animFrame = 0;
```

Comment:
Why is this a float? You are assigning a value to a float and then checking whether it == 0, == 1 etc later. That is a very very bad idea because floats can decide that when you set them to 1.0, they are actually 0.9999999 and they won't work